packages/cli/src/commands/verify.ts

```diff
@@ -14,7 +14,7 @@ const register: (program: any) => any = (program) => program
   .option('-o, --optimizer', 'enables optimizer option')
   .option('--optimizer-runs <runs>', 'specify number of runs if optimizer enabled.')
   .option('--remote <remote>', 'specify remote endpoint to use for verification')
-  .option('--api-key <key>', 'specify etherscan API key. To get one, go to: https://etherscan.io/myapikey')
+  .option('--api-key <key>', 'specify etherscan API key. To get one, go to: https://etherscancom.freshdesk.com/support/solutions/articles/35000022163-i-need-an-api-key')
```

Comment:
These change should be reverted. The correct URL is `https://etherscan.io/myapikey` (as it is in `master`)